documentation/sphinx/source/backups.rst

```diff
@@ -218,14 +218,17 @@ The ``start`` subcommand is used to start a backup.  If there is already a backu
 
 ::
 
-   user@host$ fdbbackup start [-t <TAG>] -d <BACKUP_URL> [-z] [-s <DURATION>] [-w] [-k '<BEGIN>[ <END>]']...
+   user@host$ fdbbackup start [-t <TAG>] -d <BACKUP_URL> [-z] [-s <DURATION>] [-p] [-w] [-k '<BEGIN>[ <END>]']...
 
 ``-z``
   Perform the backup continuously rather than terminating once a restorable backup is achieved.  Database mutations within the backup's target key ranges will be continuously written to the backup as well as repeated inconsistent snapshots at the configured snapshot rate.
 
 ``-s <DURATION>`` or ``--snapshot_interval <DURATION>``  
   Specifies the duration, in seconds, of the inconsistent snapshots written to the backup in continuous mode.  The default is 864000 which is 10 days.
 
+``-p`` or ``--partitioned_log``
+  Specifies the backup uses the partitioned mutation logs generated by backup workers. The default is to use non-partitioned mutation logs generated by backup agents.
```

Comment:
Better to say which FDB version this option starts to support at which level (experimental, or production-ready).

Reply:
If this feature is considered experimental still, then I think at this stage the `-p` shorthand should not be included yet, and the long form option should be `--partitioned_log_experimental`

Reply:
(or `--partitioned_log_beta` or some other appropriate name)

Reply:
Done. Removed `-p` and used `--partitioned_log_experimental` only.